cmd/client/command/visitor.go

```diff
@@ -0,0 +1,78 @@
+package command
+
+import (
+	"bufio"
+	"io"
+	"strings"
+
+	"k8s.io/apimachinery/pkg/util/yaml"
+)
+
+// VisitorFunc executes visition logic
+type VisitorFunc func([]spec)
```

Comment:
Propose to change the prototype to `type VistorFunc func(*spec)`, that's do the for loop in function `Visit`.
This could increase the reusability of the `VistorFunc` (because its parameter becomes more general) and avoid the for loop in every `VistorFunc`.